src/pages/index.js

```diff
@@ -70,31 +141,60 @@ function handleImageClick(name, link) {
   newImagePopup.open({ link, name });
 }
 
-const cardSection = new Section(
-  {
-    items: cardData,
-    renderer: renderCard,
-  },
-  ".cards__list"
-);
-
 function renderCard(cardData) {
-  const card = new Card(cardData, "#card-template", handleImageClick);
+  const card = new Card(
+    cardData,
+    "#card-template",
+    handleImageClick,
+    handleDeleteClick,
+    handleLikeClick
+  );
   const cardElement = card.getView();
```

Comment:
**Could be improved**

It's highly recommended to make a separate function `createCard` for creating a card not to copy-paste the code if you want to add with `append`

```js		

function createCard(item) {
    // here you create a card
  return cardElement
}

```


So, here will be


```js

  const cardElement = createCard(cardData) 

```